integrations/pgvector/src/haystack_integrations/document_stores/pgvector/document_store.py

```diff
@@ -129,6 +130,12 @@ def __init__(
             `"hnsw"`. You can find more information about this parameter in the
             [pgvector documentation](https://github.com/pgvector/pgvector?tab=readme-ov-file#hnsw).
         :param keyword_index_name: Index name for the Keyword index.
+        :param connection_param_kwargs: A dictionary of parameters for the PostgreSQL connection.
+            If you prefer not to use the `connection_string`, you can specify connection parameters here.
```

Comment:
```suggestion
            You can specify individual connections parameters here instead of a  `connection_string`
```

Reply:
I would suggest moving the `connection_param_kwargs` below the `connection_string` parameter, since they are both related and then also move the `connection_param_kwargs` docstring also right after the `connection_string`.

Reply:
Good call out @davidsbatista. Thank you for the review. I've made the changes in the latest commit.